assets/questions/stringToWords.js

```diff
@@ -0,0 +1,117 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for String to Dict.
+ */
+
+globalData.questions['stringToWords'] = {  // eslint-disable-line dot-notation
+  title: 'Split String to Words',
+  starterCode: {
+    python:
+`def stringToWords(lst):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def returnOriginalString(cls, s):
+        return s
+
+    @classmethod
+    def processLongStringToStringAndDict(cls, s):
+      spl = s.split(":")
```

Comment:
split_string. Feel free to specify your variable names so that they're as readable as possible -- we don't need to go for brevity, here.